packages/boot/test/fast-usdc/fast-usdc.test.ts

```diff
@@ -134,21 +135,84 @@ test.serial('writes status updates to vstorage', async t => {
   const wallet = await wd.provideSmartWallet(
     'agoric144rrhh4m09mh7aaffhm6xy223ym76gve2x7y78',
   );
-  const submitMockEvidence = (mockEvidence: CctpTxEvidence) =>
-    wallet.sendOffer({
-      id: 'submit-mock-evidence',
-      invitationSpec: {
-        source: 'agoricContract',
-        instancePath: ['fastUsdc'],
-        callPipe: [['makeTestPushInvitation', [mockEvidence]]],
-      },
-      proposal: {},
-    });
-  const mockEvidence1 = MockCctpTxEvidences.AGORIC_PLUS_OSMO();
-  const mockEvidence2 = MockCctpTxEvidences.AGORIC_PLUS_DYDX();
+  await wallet.sendOffer({
+    id: `submit-mock-evidence-dydx`,
+    invitationSpec: {
+      source: 'agoricContract',
+      instancePath: ['fastUsdc'],
+      callPipe: [
+        ['makeTestPushInvitation', [MockCctpTxEvidences.AGORIC_PLUS_DYDX()]],
+      ],
+    },
+    proposal: {},
+  });
 
-  await submitMockEvidence(mockEvidence1);
-  await submitMockEvidence(mockEvidence2);
+  const doc = {
+    node: `fastUsdc.status`,
+    owner: `the statuses of fast USDC transfers identified by their tx hashes`,
+  };
+  await documentStorageSchema(t, storage, doc);
+});
+
+test.serial('makes usdc advance', async t => {
+  const { walletFactoryDriver: wd, storage, agoricNamesRemotes } = t.context;
+  const oracles = await Promise.all([
+    wd.provideSmartWallet('agoric144rrhh4m09mh7aaffhm6xy223ym76gve2x7y78'),
+    wd.provideSmartWallet('agoric19d6gnr9fyp6hev4tlrg87zjrzsd5gzr5qlfq2p'),
+    wd.provideSmartWallet('agoric19uscwxdac6cf6z7d5e26e0jm0lgwstc47cpll8'),
+    wd.provideSmartWallet('agoric1krunjcqfrf7la48zrvdfeeqtls5r00ep68mzkr'),
+    wd.provideSmartWallet('agoric1n4fcxsnkxe4gj6e24naec99hzmc4pjfdccy5nj'),
+  ]);
+  await Promise.all(
+    oracles.map(wallet =>
+      wallet.sendOffer({
+        id: 'claim-oracle-invitation',
+        invitationSpec: {
+          source: 'purse',
+          instance: agoricNamesRemotes.instance.fastUsdc,
+          description: 'oracle operator invitation',
+        },
+        proposal: {},
+      }),
+    ),
+  );
+  await eventLoopIteration();
+
+  const usdc = agoricNamesRemotes.vbankAsset.USDC.brand;
```

Comment:
```suggestion
  /** @type {Brand<'nat'>} */
  const usdc = agoricNamesRemotes.vbankAsset.USDC.brand;
```

if necessary, add a `@ts-expect-error` here.

Reply:
Done with `as` (since it's a ts file)